bots/zephyr_mirror_backend.py

```diff
@@ -204,8 +212,7 @@ def update_subscriptions():
     classes_to_subscribe = set()
     for stream in public_streams:
         zephyr_class = stream.encode("utf-8")
-        if (options.shard is not None and
-                not hashlib.sha1(zephyr_class).hexdigest().startswith(options.shard)):
+        if (options.shard is not None and not hashlib.sha1(zephyr_class).hexdigest().startswith(options.shard)):
```

Comment:
For this PR I prefer that @anirudhjain75 not make any changes except those specifically adding type annotations, which is why I told him to return these lines to the way they were originally.

(The parentheses are needed in the original because Python doesn't terminate statements at newlines between parentheses.)